src/transaction_values.ts

```diff
@@ -83,113 +110,166 @@ async function resolveValue(t: Transaction, receipt: any, events: number) {
       t.gas = gas;
     }
 
-    receipt?.logs.map(async (log) => {
-      const parsedLog = await parseSeaportLog(log);
-
-      if (
-        parsedLog &&
-        parsedLog.tokenId == t.token_id &&
-        areEqualAddresses(parsedLog.contract, t.contract)
-      ) {
-        t.royalties = parsedLog.amount;
-        t.value = parsedLog.totalAmount ? parsedLog.totalAmount : t.value;
+    if (receipt) {
+      let totalValue = 0;
+      let totalRoyalties = 0;
+      await Promise.all(
+        receipt.logs.map(async (log) => {
+          const parsedLog = await parseSeaportLog(t, royaltiesAddress, log);
+          if (
+            parsedLog &&
+            parsedLog.tokenId == t.token_id &&
+            areEqualAddresses(parsedLog.contract, t.contract)
+          ) {
+            t.royalties = parsedLog.royaltiesAmount;
+            t.value = parsedLog.totalAmount;
+          } else {
+            if (areEqualAddresses(log.topics[0], TRANSFER_EVENT)) {
+              try {
+                const address = log.address;
+                if (!areEqualAddresses(address, LOOKS_TOKEN_ADDRESS)) {
+                  const from = resolveLogAddress(log.topics[1]);
+                  const to = resolveLogAddress(log.topics[2]);
+                  const value = resolveLogValue(log.data);
+                  if (areEqualAddresses(from, t.to_address)) {
+                    totalValue += value;
+                    if (areEqualAddresses(to, royaltiesAddress)) {
+                      totalRoyalties += value;
+                    }
+                  }
+                }
+              } catch (e) {
+                logger.error(
+                  `Error adding royalties for transaction ${t.transaction}`,
+                  e
+                );
+              }
+            }
+          }
+        })
+      );
+      if (totalValue) {
+        t.value = totalValue;
+        t.royalties = totalRoyalties;
       }
-    });
+    }
   }
 
   return t;
 }
 
-export const runValues = async () => {
+const parseSeaportLog = async (
+  t: Transaction,
+  royaltiesAddress: string,
+  log: ethers.providers.Log
+) => {
+  let seaResult;
+  try {
+    seaResult = SEAPORT_IFACE.parseLog(log);
+  } catch (err: any) {
+    // logger.error('SEAPORT PARSE ERROR', t.transaction, err);
```

Comment:
You can change it to:
```
logger.debug(`SEAPORT PARSE ERROR for transaction ${t.transactio}`, err);
```
This way it's usually not logged but you can activate this log without changing code.